### DISCUSSION TITLE
Adding review or key summary

### DISCUSSION THREAD

**Discussion by lukkiddd** [dwsmith1983/recommend-resources#23] [Ideas]
I'm thinking of adding the key takeaway or recap from each book once we've read it.

For example, we may adding 3 sections for each book as a book summary page.

1. **Who should read this book**
2. **This book will enable you to...**
3. **Top Takeaways...**

Any thoughts? @dwsmith1983

**Comment by dwsmith1983** [comment]
That would be find but how would fit in the current structure? 

Under each link there will be a description? Or make the links go to subpages that contain the book link with the information?

**Reply by lukkiddd** [reply]
I'm thinking of

- Book Recommendation (list of books)
- When clicking each book, it will go the there page with Key summary.

**Reply by dwsmith1983** [reply]
Can you create PR demoing this style so I can visualize what you mean?
